tests/jupyter/test_initialize_in_module.ipynb

```diff
@@ -164,7 +164,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.8.2"
+   "version": "3.8.0"
```

Comment:
why lower the version?

Reply:
This is just the version of the Python I used to generate the output. it's not significant/